store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -49,14 +49,6 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
     logger.info("shut down {} {}", dsConfig.host, dsConfig.database)
   }
 
-  // Return true to get all the events from the stream of updates from the data-coordinator
```

Comment:
Robert was, but the spandex secondary currently gets working copies, which seems weird.